elements/pfe-tabs/BaseTabs.ts

```diff
@@ -152,9 +164,17 @@ export abstract class BaseTabs extends LitElement {
   }
 
   override render() {
+    const classes = {
+      'inset-sm': this.inset === 'sm',
+      'inset-md': this.inset === 'md',
+      'inset-lg': this.inset === 'lg',
+      'inset-xl': this.inset === 'xl',
+      'inset-2xl': this.inset === '2xl',
+      'inset-page': this.inset === 'page'
+    };
```

Comment:
We could shorthand this to:

```suggestion
    const classes = { [`inset-${this.inset}`]: true };
```

Reply:
I really phoned that one in at the end of the day huh? 😆